ccic/data/training_data.py

```diff
@@ -48,7 +48,7 @@ def replace_zeros(data, low, high, rng):
     data = data.copy()
     mask = (data >= 0.0) * (data < high)
     low = np.log10(low)
-    high = np.log10(high)
+    high = np.log10(high) - 1.0
```

Comment:
I don't follow this change. Why is `np.log10(high) - 1.0` a better choice?

Reply:
The point with this change was to separate the small values from the remaining values hoping that this would yield better maximum a posterior estimates. Didn't turn out to work very well, but this was how the current retrieval is trained.